DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -67,8 +76,14 @@ public IActionResult EditAdminFieldStart(int customisationId, int promptNumber)
         [Route("{customisationId}/AdminFields/{promptNumber:int}/Edit")]
         public IActionResult EditAdminField(int customisationId, int promptNumber)
         {
+            if (!VerifyAdminUserCanAccessCourse(customisationId))

```

Comment:
Do we need this for POST below?

Reply:
Added the new Service filter on the post